lib/validators/omnivore.js

```diff
@@ -38,7 +40,56 @@ module.exports = function validateOmnivore(opts, callback) {
       if (size > limits.max_filesize) {
         return callback(invalid('File is larger than ' + prettyBytes(limits.max_filesize) + '. Too big to process.'));
       }
-      callback();
+      //only look at files that passed above file size test
+      files.forEach(function(f) {
+        q.defer(validateGdal, f, filetype);
```

Comment:
ok, still figuring out, how this all fits together into the bigger picture of uploads in production.
I presumed that shapefile would be the only exception, but wasn't quite sure.